### PR TITLE
nits: Fix a whole bunch of typos all over the place

### PR DESCRIPTION
**Pull request by torokati44**
I just happened to stumble upon https://github.com/crate-ci/typos referred to in the egui repo, so...

### CODE REVIEW

File: core/build_playerglobal/src/lib.rs [hunk 1]
```diff
@@ -524,7 +524,7 @@ fn write_native_table(data: &[u8], out_dir: &Path) -> Result<Vec<u8>, Box<dyn st
 fn collect_stubs(root: &Path, out_dir: &Path) -> Result<(), Box<dyn std::error::Error>> {
     let pattern = RegexBuilder::new(
         r#"
-            \b (?P<type> stub_method | stub_getter | stub_setter | stub_constructor) \s*
```
Comment (Lord-McSweeney) [inline core/build_playerglobal/src/lib.rs:527]: Is this one correct? It's in a regexp

File: core/src/avm2/globals/flash/text/engine/BreakOpportunity.as [hunk 2]
```diff
@@ -5,22 +5,22 @@
...
         // Bases line break opportunities on Unicode character properties.
         public static const AUTO:String = "auto";
-        
+
         // Treats no characters in the ContentElement object as line break opportunities.
         public static const NONE:String = "none";
-        
-        
+
+
```
Comment (Lord-McSweeney) [inline core/src/avm2/globals/flash/text/engine/BreakOpportunity.as:24]: Lines 23 and 24 here can be deleted

File: tests/tests/swfs/avm2/bitmapdata_setvector/output.txt [hunk 3]
```diff
@@ -18,7 +18,7 @@ Error: TypeError: Error #2007: Parameter rect must be non-null.
 Bitmap: [4294967295,4294967295,4294967295,4294901760,4294901760,4294901760,4278255360,4278255360,4278255360]
 
 // bData.setVector((x=0, y=0, w=3, h=3), null);
-Error: TypeError: Error #2007: Parameter imputVector must be non-null.
+Error: TypeError: Error #2007: Parameter inputVector must be non-null.
```
Comment (Lord-McSweeney) [inline tests/tests/swfs/avm2/bitmapdata_setvector/output.txt:21]: This is done to match a typo in Flash

File: core/src/avm2/globals/flash/text/engine/BreakOpportunity.as [hunk 4]
```diff
@@ -5,22 +5,22 @@
...
 package flash.text.engine
 {
-    
-    
+
+
     public final class BreakOpportunity
     {
-        // Treats all characters in the ContentElement object as line break opportunities, meaning that a line break will occur afer each character.
+        // Treats all characters in the ContentElement object as line break opportunities, meaning that a line break will occur after each character.
```
Comment (torokati44) [inline core/src/avm2/globals/flash/text/engine/BreakOpportunity.as:12]: Actually this typo is canon:
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/text/engine/BreakOpportunity.html#ALL

Still I think in a comment, we are free to fix.

File: tests/tests/swfs/avm2/bitmapdata_setvector/output.txt (hunk 3 above)
Comment (torokati44) [reply tests/tests/swfs/avm2/bitmapdata_setvector/output.txt]: Right, reverted.

File: core/src/avm2/globals/flash/text/engine/BreakOpportunity.as (hunk 2 above)
Comment (torokati44) [reply core/src/avm2/globals/flash/text/engine/BreakOpportunity.as]: Reverted whitespace-only changes in this file.

File: core/src/avm2/globals/flash/net/FileReference.as [hunk 5]
```diff
@@ -25,7 +25,7 @@ package flash.net
         }   
 
         public function get creator(): String {
-            retunr this._creator;
+            return this._creator;
```
Comment (torokati44) [inline core/src/avm2/globals/flash/net/FileReference.as:28]: "How did this ever work?!" vibes...

**Review by Dinnerbone — APPROVED** [review]

**Pull request merged**